app/pages/UcsfClientEligibilityPage/UcsfClientEligibilityPage.module.scss

```diff
@@ -64,6 +64,7 @@
 
 .eligibilitiesBtns {
   text-align: right;
+  margin-bottom: 20px;
```

Comment:
Good call out and thanks for the link! I've moved spacing management to the parent ".grid" component using `padding`. (There is only one row within that component so I cannot use gap). The `eligibilitiesBtns` children were also setting a margin-left, so I moved that to use `gap` under the auspice of the `eligibilitiesBtns` class. BTW, is there a reason you prefer padding over margin? I suppose because the padding whitespace could technically be considered part of the element's box rather than as a space outside of the element?

Reply:
> BTW, is there a reason you prefer padding over margin? I suppose because the padding whitespace could technically be considered part of the element's box rather than as a space outside of the element?

Yeah, that's basically it, but beyond it feeling "philosophically" right, there's a more pragmatic reason to favor padding over margin. When a component sets margin, it might make it more difficult for a parent component to make use of it, since the parent might want to position it in a way such that the child is closer to an element than the child's margin allows it to. It's always easy for a parent to add margin to a child (you can use [`>`](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) to more narrowly select a specific child), but it's more difficult for a parent to undo the margin of a child. If you buy into the school of thought that the parent ought to control the layout of its internals, then a child element setting margin will interfere with that control. Padding isn't the only way that a parent element can set the spacing of its children; you can also use flexbox, `gap`, grid, or even using `first/last/nth-child` selectors to add margin to a specific child, but the point is that the parent should be the one in control.

Reply:
Gotcha, thanks for the explanation!